.jenkins/Jenkinsfile-wheel-builder

```diff
@@ -0,0 +1,150 @@
+def configs = [
+    [
+        label: 'windows',
+        versions: ['py26', 'py27', 'py34', 'py35', 'py36'],
```

Comment:
193 windows 3.3 wheel downloads in the past 30 days. 192 2.6 wheel downloads. 11783 windows wheel downloads for 2.7, 3.4, 3.5, 3.6